src/pybind/mgr/cephadm/registry.py

```diff
@@ -0,0 +1,69 @@
+import json
+import logging
+import requests
+from typing import List, Dict, Tuple, TYPE_CHECKING, Set
+from pkg_resources import parse_version
+from requests import Response, RequestException
+
+from orchestrator import OrchestratorError
+
+if TYPE_CHECKING:
+    from cephadm.module import CephadmOrchestrator
```

Comment:
```suggestion
import requests
from typing import List, Dict, Tuple
from requests import Response
```

```
cephadm/registry.py:1:1: F401 'json' imported but unused
cephadm/registry.py:2:1: F401 'logging' imported but unused
cephadm/registry.py:4:1: F401 'typing.Set' imported but unused
cephadm/registry.py:5:1: F401 'pkg_resources.parse_version' imported but unused
cephadm/registry.py:6:1: F401 'requests.RequestException' imported but unused
cephadm/registry.py:8:1: F401 'orchestrator.OrchestratorError' imported but unused
cephadm/registry.py:11:5: F401 'cephadm.module.CephadmOrchestrator' imported but unused
7     F401 'json' imported but unused
```

At least in its current state